contracts/ethereum/contracts/BaseNativeToken.sol

```diff
@@ -17,13 +20,15 @@ import "./openzeppelin/SafeERC20.sol";
 * @dev Do not manually set balances without updating totalSupply, as
 * the sum of all user balances must not exceed it.
 */
-abstract contract BaseNativeToken is IERC20 {
+abstract contract BaseNativeToken is IERC20, IERC2612 {
     using SafeERC20 for IERC20;
 
     /*//////////////////////////////////////////////////////////////
                             METADATA STORAGE
     //////////////////////////////////////////////////////////////*/
 
+    bool private created_;
```

Comment:
Anyone can call the function init function, even when the contract's GovToken, so it needs to be versioned to prevent it from being called again (the GovToken constructor does this)